Regenerate code with zcbor 0.9.0 (#16974)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

DFU libraries
-------------

> * fmfu_fdev: Regenerated generated code files using zcbor 0.9.0.

Reviewer: ```suggestion
* :ref:`lib_fmfu_fdev`:

  * Regenerated the code files using zcbor v0.9.0.
```

|no_changes_yet_note|

> sdfw
> ----

Reviewer: ```suggestion
Secure Domain Firmware (SDFW) libraries
---------------------------------------
```

Reviewer: @oyvindronningstad , is this what you're referring to here?

> * sdfw_services: Regenerated generated code files using zcbor 0.9.0.

Reviewer: Hey @FrancescoSer , do we have these libraries documented somewhere?

zcbor
=====

* Updated the zcbor library to version 0.9.0.
  Full release notes at https://github.com/NordicSemiconductor/zcbor/blob/0.9.0/RELEASE_NOTES.md

>   Migration guide at https://github.com/NordicSemiconductor/zcbor/blob/0.9.0/MIGRATION_GUIDE.md

Reviewer: Links must be added through links.txt.

  Highlights:

>     * Many code generation bugfixes
> 
>     * You can now decide at run-time whether the decoder should enforce canonical encoding.
> 
>     * Allow --file-header to accept a path to a file with header contents

Reviewer: ```suggestion
    * You can now decide at runtime whether the decoder should enforce canonical encoding.
    * Allow ``--file-header`` to accept a path to a file with header contents.
    * Many code generation bug fixes.
```
